src/main/java/com/google/sps/data/UserDatastoreRepository.java

```diff
@@ -0,0 +1,130 @@
+package com.google.sps.data;
```

Comment:
Can we call this file "PersistentUserRepository.java" to keep up the same naming conventions?

Reply:
Done